tests/dmctl_basic/run.sh

```diff
@@ -178,6 +178,41 @@ function run() {
 		"\"result\": true" 2 \
 		'"worker": "worker1"' 1
 
+	echo "############################################################"
+	echo "test case for operate tasks bound to a source"
+	echo "############################################################"
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" "start-task $cur/conf/foo-task.yaml" "\"result\": true" 3
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" "start-task $cur/conf/bar-task.yaml" "\"result\": true" 3
```

Comment:
Can you add a `source3` to `bar-task.yaml` not `foo-task.yaml`? And test `pause-task/resume-task/stop-task -s source3`